site/content/docs/5.2/forms/validation.md

```diff
@@ -111,6 +111,12 @@ Custom feedback styles apply custom colors, borders, focus styles, and backgroun
 
 ## Browser defaults
 
+{{< ods-incompatibility-alert >}}
+This form variant, **with browser default validation feedback messages**, should not be used because it does not respect the Orange Design System specifications.
+
+Instead, please refer to our Boosted [Custom styles]({{< docsref "/forms/validation#custom-styles" >}}) section. You can also refer to the [Forms](https://system.design.orange.com/0c1af118d/p/88ab5b-forms/b/599459) guidelines and to the [Pages](https://system.design.orange.com/0c1af118d/p/20500e-form/b/16bb53) form examples on the Orange Design System website.
+{{< /ods-incompatibility-alert >}}
+
 Not interested in custom validation feedback messages or writing JavaScript to change form behaviors? All good, you can use the browser defaults. Try submitting the form below. Depending on your browser and OS, you'll see a slightly different style of feedback.
```

Comment:
Still think this one is a bit confusing: 'should not be used [...]' and just after 'All good, you can use [it]'

Reply:
OK dropping it from this PR

Reply:
@MewenLeHo: any suggestion to fix that?
Possibilities we identified:
1- place this callout just after the "Browser defaults" title to make the user stop reading what is below the callout. This one was agreed on.
2- modify the text "Not interested in custom [...] feedback."

Other possibility:
3- place this callout after the text "Not interested in custom [...] feedback." and just above the illustration ?

Reply:
We should maybe try the third solution since it doesn't break the Bootstrap text ?

Reply:
Let's go for this 3rd solution. I'm doing it in PR #1614.